esp-radio/src/common_adapter/common_adapter_esp32.rs

```diff
@@ -167,7 +167,7 @@ unsafe extern "C" fn __esp_radio_phy_enter_critical() -> u32 {
 /// *************************************************************************
 #[ram]
 #[unsafe(no_mangle)]
-unsafe extern "C" fn __esp_radio_phy_exit_critical(level: u32) {
```

Comment:
the idea is to prefix the "extern" functions so they don't likely collide with anything else - then the liner script will provide the real symbols

this way it's not possible to override them anymore

Reply:
I actually don't know, why, how and when I changed that. But yeah, that doesn't look right.

Reply:
Ok, I figured it out. I actually intended on deleting those files anyway...